dockerfiles/evmapp/entrypoint.sh

```diff
@@ -90,56 +100,70 @@ done
 
 if [ "${SCNODE_FORGER_RESTRICT:-}" = "true" ]; then
   if [ -z "${SCNODE_ALLOWED_FORGERS:-}" ]; then
-    echo "Error: Environment variable SCNODE_ALLOWED_FORGERS required when SCNODE_FORGER_RESTRICT=true."
+    echo "Error: Environment variable SCNODE_ALLOWED_FORGERS is required when SCNODE_FORGER_RESTRICT=true."
     sleep 5
     exit 1
   fi
 fi
 
-SCNODE_REMOTE_KEY_MANAGER_ENABLED="${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-false}"
-export SCNODE_REMOTE_KEY_MANAGER_ENABLED
-
+# Cert signing
 if [ "${SCNODE_CERT_SIGNING_ENABLED:-}" = "true" ]; then
-  if [ -z "${SCNODE_CERT_SIGNERS_SECRETS:-}" ] && [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" != "true" ]; then
-    echo "Error: When SCNODE_CERT_SIGNING_ENABLED=true either environment variable SCNODE_CERT_SIGNERS_SECRETS is required or remote key manager SCNODE_REMOTE_KEY_MANAGER_ENABLED=true is required."
-    sleep 5
-    exit 1
-  fi
-
-  if  [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" = "true" ] && [ -z "${SCNODE_REMOTE_KEY_MANAGER_ADDRESS:-}" ]; then
-    echo "Error: When SCNODE_CERT_SIGNING_ENABLED=true and SCNODE_REMOTE_KEY_MANAGER_ENABLED=true SCNODE_REMOTE_KEY_MANAGER_ADDRESS needs to be set."
-    sleep 5
-    exit 1
-  fi
 
-  if  [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" = "true" ] && [ -z "${SCNODE_REMOTE_KEY_MANAGER_REQUEST_TIMEOUT:-}" ]; then
-    echo "Error: When SCNODE_CERT_SIGNING_ENABLED=true and SCNODE_REMOTE_KEY_MANAGER_ENABLED=true SCNODE_REMOTE_KEY_MANAGER_REQUEST_TIMEOUT needs to be set."
+  # Requirement when SCNODE_CERT_SIGNING_ENABLED = true
+  if [ -z "${SCNODE_CERT_SIGNERS_SECRETS:-}" ]; then
+    echo "Error: When SCNODE_CERT_SIGNING_ENABLED=true, 'SCNODE_CERT_SIGNERS_SECRETS' variable is required to be set."
```

Comment:
This is not true right? This variable can be empty if SCNODE_REMOTE_KEY_MANAGER_ENABLED is true

Reply:
Just spoke with Luigi about it and fixing my PR. You are right on this.